### PR TITLE
r-vgam, r-zoo

### PR DESCRIPTION
**Pull request by croth1**
first trial of r packages 🎉

### CODE REVIEW

**Comment by jakirkham** [comment]
Might wanted to update issue ( https://github.com/conda-forge/staged-recipes/issues/2009 ) about these additions. @johanneskoester is trying to track R packages. 😄

File: recipes/r-vgam/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+    - lib/R/lib/
+    - lib/
+
+# Suggests: VGAMdata, MASS, mgcv
+requirements:
+  build:
+    - r-base
+    - posix                # [win]
+    - {{native}}toolchain  # [win]
+    - gcc                  # [not win]
```
Comment (croth1) [inline recipes/r-vgam/meta.yaml]: @jakirkham, the cran skeleton automatically includes gcc. Is that the way to go now in conda-forge as well?

Comment (johanneskoester) [inline recipes/r-vgam/meta.yaml]: At least for r packages yes. Also make sure to use conda-build 2.1.9, it supports the conda-forge style of r recipes, including a sha256 checksum.

**Comment by johanneskoester** [comment]
Can you remove the comment lines?

**Comment by croth1** [comment]
> There are 3 too many lines. There should be one empty line at the end of the file

A very creative way of telling that there are three empty lines at the end of the file, but there should be none.

@jakirkham

**Comment by jakirkham** [comment]
Yeah, initially it just said too many, but another conda-forge member did not like this and wanted an explicit number of lines in the message. So we have that instead now. Do you not like this or do you just find the wording funny?

**Comment by croth1** [comment]
Well, I am not a native speaker - it seems a bit gibberish to me.

Shouldn't it be:
> There are 3 empty lines at the end of the file, but there should be none.

or something like this?

**Comment by jakirkham** [comment]
Yep, you're right. Definitely reads funny. I like your rephrasing of it. Would you like to submit a fix to `conda-smithy`?

File: recipes/r-vgam/bld.bat [hunk 2]
```diff
@@ -0,0 +1,8 @@
+"%R%" CMD INSTALL --build .
+if errorlevel 1 exit 1
+
+@rem Add more build steps here, if they are necessary.
```
Comment (johanneskoester) [inline recipes/r-vgam/bld.bat]: Could you please cleanup the build scripts (remove all comments).

File: recipes/r-vgam/build.sh [hunk 3]
```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+# R refuses to build packages that mark themselves as Priority: Recommended
+mv DESCRIPTION DESCRIPTION.old
+grep -v '^Priority: ' DESCRIPTION.old > DESCRIPTION
```
Comment (johanneskoester) [inline recipes/r-vgam/build.sh]: Above three lines are not needed (only for core packages, which are already done).

**Pull request merged**